app-doc/eclass-manpages/files/eclass-to-manpage.awk

```diff
@@ -90,10 +90,9 @@ function xfail(text) {
 	_stderr_msg(text, "error (ignoring)")
 	exit(77)
 }
-
 function eat_line() {
 	ret = $0
-	sub(/^# @[A-Z]*:[[:space:]]*/,"",ret)
+	sub(/^# @.*:[[:space:]]*/,"",ret)
```

Comment:
Note: I originally created a new function called `eat_eapis`, without touching this eat_line sub, but decided to give this a try and it didn't seem to break anything with the 10 eclasses I tried it with. No knowledge about **possible** breakages, that's why it's up for review ;)

```
function eat_eapis() {
	eapiret = $0
	sub(/^# @SUPPORTED_EAPIS*:[[:space:]]*/,"",eapiret)
	getline
	return eapiret
}
```